inloop/statistics/static/statistics/vendor/css/statistics.css

```diff
@@ -0,0 +1,125 @@
+body {
```

Comment:
The CSS files seems to replicate things from our main CSS, this should be cleaned up.

Reply:
This is remnant from the initial thought to ship the statistics app as a universally applicable package. I need to re-evaluate this.